test/integration/serverless/parse-json.js

```diff
@@ -1,17 +0,0 @@
-'use strict'
```

Comment:
This file is no longer needed because the python script now performs the json prettying.